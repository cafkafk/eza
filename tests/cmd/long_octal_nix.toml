bin.name = "eza"
args = "tests/itest --long --octal-permissions"
