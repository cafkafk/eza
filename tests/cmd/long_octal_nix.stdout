0644 .rw-r--r--  0 nixbld  1 Jan  1970 a
0644 .rw-r--r--  0 nixbld  1 Jan  1970 b
0644 .rw-r--r--  0 nixbld  1 Jan  1970 c
0644 .rw-r--r--  0 nixbld  1 Jan  1970 d
0644 .rw-r--r--  0 nixbld  1 Jan  1970 e
0755 drwxr-xr-x  - nixbld  1 Jan  1970 exa
0644 .rw-r--r--  0 nixbld  1 Jan  1970 f
0644 .rw-r--r--  0 nixbld  1 Jan  1970 g
0644 .rw-r--r--  0 nixbld  1 Jan  1970 h
0644 .rw-r--r--  0 nixbld  1 Jan  1970 i
0644 .rw-r--r--  0 nixbld  1 Jan  1970 image.jpg.img.c.rs.log.png
0644 .rw-r--r-- 19 nixbld  1 Jan  1970 index.svg
0644 .rw-r--r--  0 nixbld  1 Jan  1970 j
0644 .rw-r--r--  0 nixbld  1 Jan  1970 k
0644 .rw-r--r--  0 nixbld  1 Jan  1970 l
0644 .rw-r--r--  0 nixbld  1 Jan  1970 m
0644 .rw-r--r--  0 nixbld  1 Jan  1970 n
0644 .rw-r--r--  0 nixbld  1 Jan  1970 o
0644 .rw-r--r--  0 nixbld  1 Jan  1970 p
0644 .rw-r--r--  0 nixbld  1 Jan  1970 q
0755 drwxr-xr-x  - nixbld  1 Jan  1970 vagrant
